src/components/Logger/index.js

```diff
@@ -190,12 +190,15 @@ export class Logger extends React.Component {
         )
     }
 
-    getDate(p, prevDateHH, prevDate) {
+    getDate(p, prevDateHH, prevDate, truncate = false) {
```

Comment:
Can you change the variable name to something meaningful or add a jsdoc commment?
I think that variables shouldn't be abbreviated with single letters as it's totally unclear what they mean.

Reply:
I agree. I just have a habit of not refactoring code I've not written (probably due to my work on maintenance, hehe). Done now.